src/main/java/org/folio/circulation/domain/FeeFine.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.circulation.domain;
+
+import io.vertx.core.json.JsonObject;
+
+public class FeeFine {
+  private final JsonObject representation;
+
+  public FeeFine() {
+    representation = null;
+  }
```

Comment:
Under what circumstances would a fee / fine have no state?

Reply:
Removed this constructor along with JSON representation.